wagtail/contrib/frontend_cache/backends.py

```diff
@@ -100,7 +100,7 @@ def purge_batch(self, urls):
 
         except requests.exceptions.HTTPError as e:
             for url in urls:
-                logger.error("Couldn't purge '%s' from Cloudflare. HTTPError: %d %s", url, e.response.status_code, e.message)
+                logger.error("Couldn't purge '%s' from Cloudflare. HTTPError: %d", url, e.response.status_code)
```

Comment:
Any reason not to use `logging.exception` instead of `logger.error`? Similarly to `logger.error` it will log with the `ERROR` log level, but will also add exception info/traceback.

Reply:
The traceback doesn't seem like particularly useful information to me, and I feel that it's just adding noise to the logs:

```
ERROR:root:Couldn't purge 'http://localhost/events/' from Cloudflare. HTTPError: 500
Traceback (most recent call last):
  File "/vagrant/wagtail/wagtail/contrib/frontend_cache/backends.py", line 89, in purge_batch
    headers=headers,
  File "/usr/lib/python3.6/unittest/mock.py", line 939, in __call__
    return _mock_self._mock_call(*args, **kwargs)
  File "/usr/lib/python3.6/unittest/mock.py", line 995, in _mock_call
    raise effect
```
Surely the important information here is that Cloudflare returned a 500 error - I don't think we gain anything from knowing that it happened on line 89 of backends.py...?

Reply:
Traceback is not very useful in tests, but without logging the traceback, how do you know in a real project what is failing? Is it Wagtail's default invalidation functionality or your [custom logic](http://docs.wagtail.io/en/v2.4/reference/contrib/frontendcache.html#invalidating-index-pages) with `PurgeBatch`? Imagine receiving a `400 Bad Request` without a traceback here given that you have custom logic in your project.

Reply:
Fair enough! Changed to `logging.exception` in 2f3b47a3dfe77a2e9c7c191a135b64baecc50e61.